data/resume.yml

```diff
@@ -72,96 +76,93 @@ summary_links:
   - text: Technical Debt (Martin Fowler)
     link: http://martinfowler.com/bliki/TechnicalDebt.html
 brief_summary: |
-    Deeply experienced in software design and development methodologies,
-    especially related to the disciplines of software quality, web
-    applications, user experience, and team building. Presently working at the intersection of Rails, GraphQL, and React, developing modern web applications with fast response times.
+    Deeply experienced in software architecture, design, and development methodologies,
+    especially related to the disciplines of web applications, user experience, team
+    building, and processes. Working at the intersection of Rails, GraphQL, and React,
```

Comment:
> Working at the intersection of Rails, GraphQL, and React

This stands out and gave me a clear impression of your skills. Something I can share with others.

> developing modern web applications with fast response times.

^^ same here. Maybe put these at the top of the summary?

Reply:
yes.. don't bury the lead